### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tcyrus**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,17 +24,19 @@ Example:
 ```
 gssh -f file -- -o ConnectTimeout=30 -o BatchMode=yes id
 ```
+
 gssh supports streaming support. Useful for tailing logs across multiple machines etc
 
-```
+```bash
```
Comment (xaviershay) [inline README.md]: why here and nowhere else?

Doesn't appear to affect the output anyways.

**Comment by xaviershay** [comment]
Thank you! Looks good aside from comment about `bash` marker.

**Comment by tcyrus** [comment]
@xaviershay, I removed the `bash` markers

**Comment by xaviershay** [comment]
thanks!

**Pull request merged**